js/pulse.js

```diff
@@ -673,10 +714,13 @@
             this.tryInitAdPlayer = function(){
                 var flashVersion = getFlashVersion().split(',').shift();
 
-                if(this.ui && adModuleJsReady) {
-
+                if(this.ui && adModuleState == AD_MODULE_STATE.Ready) {
```

Comment:
We should prefer the use of `===` over `==` as this will avoid any unwanted implicit conversion that the interpreter may perform.